### PR TITLE
Fix navigation for upstream UI changes.

### PR DESCRIPTION
**Pull request by amavinag**
* settings panel moved to top right in the upstream.
    added quick fix to unblock upstream smoke tests.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 67931a41bc68ddee8d56aae468815839cef70344:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/web_ui/menu.py [hunk 1]
```diff
@@ -439,16 +439,19 @@ def f(_):
 
         def f2(_):
             args = [toplevel, secondlevel, thirdlevel]
-            loc = cls.ROOT
-            for arg in args:
-                if arg:
-                    if arg.startswith("/"):
-                        loc_to_use = cls.NAMED_LEV_HREF
+            if version.current_version() == 'master' and toplevel == 'Settings':
```
Comment (psav) [inline cfme/web_ui/menu.py]: I think I'd actually prefer to handle this outside of "menu". This is not related to the main "menu" anymore.....I try to knock something up.

Comment (jkrocil) [inline cfme/web_ui/menu.py]: Although this is good as a quickfix, I agree with Pete - this should be handled elsewhere, not in `cfme.web_ui.menu`.

Comment (amavinag) [inline cfme/web_ui/menu.py]: sure @psav @jkrocil I'm looking at dashboard and other pages to move this out of menu.py

**Comment by psav** [comment]
I worked up #3174 to try and help simplify things a little, I think it is preferred
